docker-container/README.md

```diff
@@ -0,0 +1,61 @@
+# Based On:
+
+[https://github.com/alexcheng1982/docker-magento](https://github.com/alexcheng1982/docker-magento)
+
+With M1 being an older service there is no official support from the platform. Through my research this was the best overall solution giving a full breakdown of the process.
+
+# Dependencies
+
+## ngrok
+
+[https://ngrok.com/](https://ngrok.com/pricing)
+[https://ngrok.com/pricing](https://ngrok.com/pricing)
+
+In order for your docker container to communicate with the webhook and shipping & tax url set on [merchant.bolt.com](http://merchant.bolt.com/), one must set up an ngrok account to expose your store's docker ports to a public IP address. In order to do this one must have at least the Basic plan with ngrok to reserve a domain for themselves. 
```

Comment:
let's link to merchant-sandbox.bolt.com instead - I think primary use case is develop/test